README.md

```diff
@@ -1,22 +1,97 @@
-# Q Election Votes
-
-[![Greenkeeper badge](https://badges.greenkeeper.io/nzzdev/Q-election-votes.svg)](https://greenkeeper.io/)
+# Q Election Votes [![Build Status](https://travis-ci.com/nzzdev/Q-election-votes.svg?branch=dev)](https://travis-ci.com/nzzdev/Q-election-votes) [![Greenkeeper badge](https://badges.greenkeeper.io/nzzdev/Q-election-votes.svg)](https://greenkeeper.io/)
 
 **Maintainer**: [benib](https://github.com/benib)
 
 Q election votes is one tool of the Q toolbox to display results of parliamentary elections.
+Test it in the demo: https://editor.q.tools
+
+## Table of contents
+
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Development](#development)
+- [Testing](#testing)
+- [Tool implementation details](#tool-implementation-details)
+- [Features](#features)
+- [Options](#options)
+
+## Installation
+
+```bash
+$ nvm use
+$ npm install
+$ npm run build
+```
+
+## Configuration
+
+No configuration is needed for this tool.
+
+## Development
+
```

Comment:
@philipkueng did the development section a bit different in q-table -> https://github.com/nzzdev/Q-table/tree/feat-documentation#development

## Development

Install the [Q cli](https://github.com/nzzdev/Q-cli) and start the Q dev server:

```
$ Q server
```

Run the Q tool:
```
$ node index.js
```